app/src/main/java/org/sopt/androidseminar_week2/view/HomeActivity.kt

```diff
@@ -0,0 +1,79 @@
+package org.sopt.androidseminar_week2.view
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import org.sopt.androidseminar_week2.data.RepositoryListInfo
+import org.sopt.androidseminar_week2.databinding.ActivityHomeBinding
+
+class HomeActivity : AppCompatActivity() {
+    private lateinit var binding : ActivityHomeBinding
+    private val logTag = "logTag"
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityHomeBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        inputRepoData()
+        moreButtonOnClickListener()
+    }
+
+    private fun inputRepoData() {
+        val repositoryListAdapter = RepositoryListAdapter()
+        binding.rvRepositoryList.adapter = repositoryListAdapter
+
+        repositoryListAdapter.setRepositoryList(
+                listOf(RepositoryListInfo(
+                repoName = "이ㅣㅣㅣㅣㅣㅣㅣㅣㅣㅣㅣㅣㅣㅣㅣㅣㅣㅣㅣㅣㅣㅣㅣㅣㅣㅣ름",
+                repoDesc = "설ㄹㄹㄹㄹㄹㄹㄹㄹㄹㄹㄹㄹㄹㄹㄹㄹㄹㄹㄹㄹㄹㄹㄹㄹㄹㄹㄹㄹㄹ명",
+                repoLang = "언ㄴㄴㄴㄴㄴㄴㄴㄴㄴㄴㄴㄴㄴㄴㄴㄴㄴㄴㄴㄴㄴㄴㄴㄴㄴㄴㄴㄴㄴ어"))
+        )
+        for (i in 1 .. 10) {
+            repositoryListAdapter.setRepositoryList(
+                    listOf(RepositoryListInfo(
+                            "Repository Name".plus(i),
+                            "Repository Description".plus(i),
+                            "Repository Language".plus(i))))
+        }
```

Comment:
이것 역시 코틀린 함수형 API로!!

```suggestion
        val repositoryList = mutableListOf<RepositoryListInfo>()
        (1..10).forEach(
            repositoryList.add(RepositoryListInfo(
                            "Repository Name".plus(it),
                            "Repository Description".plus(it),
                            "Repository Language".plus(it)))
        )
```

Reply:
아하 아직 코틀린이 익숙하지 않아서 돌아가는 대로 작성했더니..ㅋㅋㅋㅋㅋ 이 부분 조금 더 공부해보고 수정할게